demo-projects/blog/package.json

```diff
@@ -35,7 +35,7 @@
     "@keystonejs/oembed-adapters": "^5.1.4",
     "apollo-cache-inmemory": "^1.6.6",
     "apollo-client": "^2.6.10",
-    "apollo-upload-client": "^13.0.0",
+    "apollo-upload-client": "^14.1.0",
```

Comment:
Upgrading these versions of `apollo-upload-client` so that all versions in the monorepo are the same